messaging/messaging.cc

```diff
@@ -94,15 +104,19 @@ PubSocket * PubSocket::create(Context * context, std::string endpoint, bool chec
 
 Poller * Poller::create(){
   Poller * p;
+#ifdef CEREAL_FAKE
   if (messaging_use_fake()) {
     p = new FakePoller();
   } else {
+#endif
     if (messaging_use_zmq()){
       p = new ZMQPoller();
     } else {
       p = new MSGQPoller();
     }
+#ifdef CEREAL_FAKE
   }
+#endif
```

Comment:
this is not readable

Reply:
Should be better now - made the conditional blocks more self-contained.